js/clipboardToggleAll.js

```diff
@@ -0,0 +1,39 @@
+(function ($)
+  {
+    // Toggle buttons
+    var clickClipButton = function (node, selectAll) {
+      $button = $(node);
+
+      // if 'none' was chosen, click any button that has 'added' on
+      if (selectAll == false && $button.hasClass('added')) {
+        $button.click();
+      }
+      // if 'all' was chosen, don't click the button if 'added' is on
+      else if (selectAll == true && !$button.hasClass('added')) {
+        $button.click();
+      }
+    };
+
+    // Convenience wrapper in jQuery
+    $.fn.clickClipButton = function(clicked) {
+      this.each(function () {
+        clickClipButton.apply(null, [this, clicked]);
+      });
+    };
+
+    // get all affected clipboard buttons and select all or none based on
+    // item clicked.
+    $(document).ready(function() {
+
+      var $area = $('#clipboardButtonNode');
+
```

Comment:
Good idea!   I have added code to do find and assign the buttons inside $(document).ready. just after $area is assigned.